auth-web/src/composables/affiliations-factory.ts

```diff
@@ -154,6 +154,16 @@ export const useAffiliations = () => {
     )
   }
 
+  const nameRequestType = (business: Business): string => {
+    if (isNameRequest(business)) {
+      const nrType: string = NrRequestType[business.nameRequest?.entityTypeCd]
+      if (nrType) {
+        return '\u2014 ' + nrType
```

Comment:
Here's how Filings UI does the dash, fwiw (it's self-documenting):

https://github.com/bcgov/business-filings-ui/blob/fafe0c3af7da84fa91f3677f785cfe5c7f0adb8b/src/stores/rootStore.ts#L228

Reply:
Ah that's much cleaner, I'll use that.